.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
I'd prefer to have IDE specific files to ignore in the user's global git settings (see https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer), because different users might use different IDEs.